### DISCUSSION TITLE
Skip plugin installation during tofu init

### DISCUSSION THREAD

**Discussion by marsskop** [opentofu/opentofu#2198] [Q&A] [answered]
Hello folks.

Once, terraform had an option `-get-plugins=false` during `terraform init` that effectively skipped plugin installation. That option was deprecated on v0.15 (https://developer.hashicorp.com/terraform/cli/commands/init#get-plugins-false). How would you emulate this behaviour now?

My case is such that I don't have providers remotely or locally, nor do I need them. I just want to manipulate the state, aka with `terraform state pull`, so I just want to run the backend initialisation part. So far I can do that by ignoring the errors OpenTofu throws when it fails to resolve provider packages, and just running `tofu state pull` after the init fails (but succeeds in backend initialisation).

Thanks for any help!

**Comment by ghost** [comment, marked as answer]
Hey @marsskop I do not think this is possible today, so I created an issue from this discussion: https://github.com/opentofu/opentofu/issues/2215

**Reply by marsskop** [reply]
Thank you! Would love to see such a feature.
